docker/mixer-adapter/Dockerfile

```diff
@@ -20,4 +20,6 @@ FROM scratch
 
 COPY ./target/mixer-adapter /mixer-adapter
 
+ENV SP_SERVER_URL "http://wso2sp-worker.cellery-system.svc.cluster.local:9091"
```

Comment:
We don't need to add this here I think. This is something we can directly est from the deployment yaml. The default value we are setting is anyway a Kubernetes specific value.